lib/services/database.go

```diff
@@ -1240,3 +1240,10 @@ const (
 	// labelResourceGroup is the label key for the Azure resource group name.
 	labelResourceGroup = "resource-group"
 )
+
+const (
+	// labelProjectID is the label key for GCP project ID.
+	labelProjectID = "project-id"
+	// labelLocation is the label key for GCP location.
+	labelLocation = "location"
+)
```

Comment:
Feels strange that these are defined in `database.go` but used by Kube discovery.

Reply:
Also, I think these label names need to be more specific. E.g. `teleport.dev/gcp/project-id`. Same for other clouds.

Reply:
I do agree with `teleport.dev/gcp/project-id`  since it's much cleaner and simpler to understand. But the DB auto-discovery already uses labels without cloud prefixes for Azure and AWS and we might break the user's RBAC if they are based on those labels